### PR TITLE
Add log-path documentation for grndb

### PR DESCRIPTION
**Pull request by kenhys**

### CODE REVIEW

File: doc/source/reference/executables/grndb.rst [hunk 1]
```diff
@@ -94,6 +94,18 @@ Here is an example that checks only ``Entries.name`` column::
 
   % grndb check --target Entries.name /var/lib/groonga/db/db
 
+``--log-path``
+""""""""""""""
+
+.. versionadded:: 7.0.4
+
+It specifies a path of groonga log.
```
Comment (kou) [inline doc/source/reference/executables/grndb.rst]:
```text
groonga -> ``grndb``
```

File: doc/source/reference/executables/grndb.rst [hunk 2]
```diff
@@ -94,6 +94,18 @@ Here is an example that checks only ``Entries.name`` column::
...
+
+Here is an example that specifies grndb log::
```
Comment (kou) [inline doc/source/reference/executables/grndb.rst]:
```text
grndb -> ``grndb``
```

File: doc/source/reference/executables/grndb.rst (hunk 1 above)
Comment (kenhys) [inline doc/source/reference/executables/grndb.rst]: I'll fix it.

**Comment by kenhys** [comment]
* fix markup about ``grndb``.
* fix "groonga log" to "grndb log".

**Comment by kou** [comment]
Merged.

**Pull request merged**